### PR TITLE
Created css for cards, labels, headings

### PR DESCRIPTION
**Pull request by fragsalat**
- Implemented styles for cards, labels, headings and new shadow variables
- splitted utils/_typography.scss into separate files and moved it to atoms

### CODE REVIEW

File: molecules/_dropdown.scss [hunk 1]
```diff
@@ -110,7 +110,7 @@ $dropdown-background-color-accent: $earl-gray !default;
     display: block;
     .dropdown-container {
       @include dropdown-container($dropdown-width, $dropdown-background-color, $dropdown-shadow);
-      &.wide {
+      &.mod-wide {
```
Comment (fokusferit) [inline molecules/_dropdown.scss]: what is the mod for ? some sort of naming ? If yes, we should document it somewhere, e.g. COntributing.md or Wiki, Readme

File: utils/variables/_shadows.scss [hunk 2]
```diff
@@ -17,13 +17,10 @@ $elevation-shadows: (
...
-$paper-shadow-1: 0 1px 3px rgba($black, 0.12),
+$paper-shadow-1: inset 0 2px 3px rgba($asphalt, 0.30);
+$paper-shadow-2: 0 1px 3px rgba($black, 0.12),
                  0 1px 2px rgba($black, 0.24);
-$paper-shadow-2: 0 3px 6px rgba($black, 0.16),
+$paper-shadow-3: 0 3px 6px rgba($black, 0.16),
                  0 3px 6px rgba($black, 0.23);
-$paper-shadow-3: 0 15px 12px rgba($black, 0.22),
-                 0 19px 38px rgba($black, 0.30);
-$paper-shadow-4: inset 0 2px 3px rgba($asphalt, 0.30);
```
Comment (fokusferit) [inline utils/variables/_shadows.scss]: Are you confident that no one is using them in a component?

**Comment by fokusferit** [comment]
Btw. The last 3 commits are not compliant to your message syntax. Does it affect your changelog process? I guess as we don't have one yet, it is fine ?

a) It needs an interactive rebase -> preferred.
b) squash commit -> lost history not my first choice.

File: molecules/_dropdown.scss (hunk 1 above)
Comment (fragsalat) [inline molecules/_dropdown.scss]: We aligned on a naming convention before some months.
State classes are prefixed by an `is-` like `is-active` or `is-hovered` and modifier classes which changes the default behavior are prefixed by mod- like `mod-large`

File: utils/variables/_shadows.scss (hunk 2 above)
Comment (fragsalat) [inline utils/variables/_shadows.scss]: Yes. I introduced the variables 2 PR's before^^

**Comment by fragsalat** [comment]
@fokusferit Commits which doesn't follow the pattern are just ignored. This is useful when you have to fix something afterwards^^

**Pull request merged**